homeassistant/components/denonavr/__init__.py

```diff
@@ -57,6 +64,19 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
```

Comment:
```suggestion
    await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
    use_telnet = entry.options.get(CONF_USE_TELNET, DEFAULT_USE_TELNET)
```

Reply:
Addressed in https://github.com/home-assistant/core/pull/85980/commits/f5e8b057d1716a236dcc1062734a5688d40a725e